homeassistant/components/geocaching/const.py

```diff
@@ -0,0 +1,78 @@
+"""Constants for the Geocaching integration."""
+from datetime import timedelta
+
+from homeassistant.const import (
+    ATTR_DEVICE_CLASS,
+    ATTR_ICON,
+    ATTR_NAME,
+    ATTR_STATE,
+    ATTR_UNIT_OF_MEASUREMENT,
+)
+
+ATTR_DEFAULT_ENABLED = "default_enabled"
+ATTR_MEASUREMENT = "measurement"
+DOMAIN = "geocaching"
+ATTR_SECTION = "section"
+
+OAUTH2_AUTHORIZE_URL = "https://staging.geocaching.com/oauth/authorize.aspx"
+OAUTH2_TOKEN_URL = "https://oauth-staging.geocaching.com/token"
+API_ENDPOINT_URL = "https://staging.api.groundspeak.com"
```

Comment:
staging?

Reply:
Yeah. We need to adjust this when we received the production keys. I keep this one open until we have them.